### PR TITLE
ColorCmdsWidget() name refactor and MayaColorSlider() widget class added

### PR DESCRIPTION
**Pull request by andrewsilke**
some changes to ColorCmdsWidget() for better naming, 
added MayaColorSlider() an embedded Maya color widget from cmds.colorSliderGrp

### CODE REVIEW

File: zoo/libs/maya/qt/cmdswidgets.py [hunk 1]
```diff
@@ -207,3 +212,220 @@ def addActionList(self, modes, mouseButton=QtCore.Qt.RightButton):
...
+
+        # 1) Create the colorSliderGrp """
+        window = cmds.window()
+        # color_slider_name
+        #  width=1 is the pixel width of the slider which is hidden,
+        #  columnWidth specifies col 1 width from the kwargs
+        color_slider_name = cmds.colorSliderGrp(width=1, columnWidth=[1, self.colorWidth])
+
+        # 2) Find the colorSliderGrp widget
+        self._color_slider_obj = om1.MQtUtil.findControl(color_slider_name)
```
Comment (dsparrow27) [inline zoo/libs/maya/qt/cmdswidgets.py]: pretty sure we have functionfor this `mayaui.toQtObject(color_slider_name)`

File: zoo/libs/maya/qt/cmdswidgets.py [hunk 2]
```diff
@@ -207,3 +212,220 @@ def addActionList(self, modes, mouseButton=QtCore.Qt.RightButton):
...
+            self._slider_widget = self._color_slider_widget.findChild(QtWidgets.QWidget, "slider")
+            if self._slider_widget:
+                self._slider_widget.hide()
+
+            self._color_widget = self._color_slider_widget.findChild(QtWidgets.QWidget, "port")
+            cmds.colorSliderGrp(self._getFullName(), e=True, changeCommand=partial(self._onColorChanged))
+
+        cmds.deleteUI(window, window=True)
+
+    def _getFullName(self):
```
Comment (dsparrow27) [inline zoo/libs/maya/qt/cmdswidgets.py]: mayaui.fullName

File: zoo/libs/maya/qt/cmdswidgets.py [hunk 3]
```diff
@@ -207,3 +212,220 @@ def addActionList(self, modes, mouseButton=QtCore.Qt.RightButton):
...
+
+        :param modes: a list of menu modes (tuples) eg [("icon1", "menuName1"), ("icon2", "menuName2")]
+        :type modes: list(tuple(str))
+        :param mouseButton: the mouse button clicked QtCore.Qt.LeftButton, QtCore.Qt.RightButton, QtCore.Qt.MiddleButton
+        :type mouseButton: QtCore.Qt.ButtonClick
+        """
+        # only add the action list (menu items) to the label, as the line edit uses the same menu
+        if self.label:
+            self.label.addActionList(modes, mouseButton=mouseButton)
+        # else:
```
Comment (dsparrow27) [inline zoo/libs/maya/qt/cmdswidgets.py]: remove

File: zoo/libs/maya/qt/cmdswidgets.py (hunk 2 above)
Comment (dsparrow27) [reply zoo/libs/maya/qt/cmdswidgets.py]: also dont use the `get` prefix , totally useless description

Comment (andrewsilke) [reply zoo/libs/maya/qt/cmdswidgets.py]: Yeah, I've stopped using get as per Qt, though I still have loads of function in the lighting tools with get, this is from Chris's code, will clean it up.

File: zoo/libs/maya/qt/cmdswidgets.py (hunk 3 above)
Comment (andrewsilke) [reply zoo/libs/maya/qt/cmdswidgets.py]: done

File: zoo/libs/maya/qt/cmdswidgets.py (hunk 1 above)
Comment (andrewsilke) [reply zoo/libs/maya/qt/cmdswidgets.py]: yeah found it, and it works, great.  Though looks like I still need the control pointer for fullName? Unless there's another way?
```
def fullName(self):
    return om1.MQtUtil.fullName(long(self._color_slider_obj))
```

**Comment by andrewsilke** [comment]
should be good to go

**Pull request merged**